src/index.js

```diff
@@ -111,6 +111,9 @@ export const createGlobalState = (initialState) => {
 };
 
 export const createStore = (reducer, initialState, enhancer) => {
+  if (!initialState) {
+    initialState = reducer(undefined, { type: undefined });
```

Comment:
@jarlah Does your reducer accept undefined action? I'd like to simply do this.
```suggestion
    initialState = reducer();
```